training/culture-and-process/CULTURE.md

```diff
@@ -0,0 +1,48 @@
+Culture in DP
+===========================
+Our culture and how we work is important for us to create an inclusive, empathetic and successful environment that works for everybody. 
+
+
+### Principles
+
+#### Inclusion
+We believe people from different backgrounds, with different identities and experiences, make our product, and our team, better. 
+
+We know a happy, comfortable and safe environment where our people are free to express themselves and their ideas means we create the best experiences for our users. 
+
+#### User first
+We put out users at the heart of everything we do. This includes external and internal users. We know our users come from everywhere with a range of expertise and needs which helps to create a better product for all.
+
+We follow [GDS design principles](https://www.gov.uk/guidance/government-design-principles#start-with-user-needs). 
+
+#### Open communication and collaboration
+We code in the open. This means most of our code is publicly available on Github and it hosts a lot of our coding discussions.
+
+We encourage as many conversations and calls as possible to happen in open slack channels so that they are inclusive to all and can act as a source of information in the future. To let others know your concentrating on a task or that you'd like to minimise distractions use the headphones emoji 🎧 .
+
+
+Whichever platform we use to communicate we use open and inclusive language. 
+
+#### Self care
+We believe in everyone being happy and well and we know that everyone has different personal needs, circumstances and responsibilities. This means healthy and flexible working patterns and being mindful of others availability when scheduling events and communicating.
+
+We encourage a healthy work life balance and as a team discuss our work load openly, especially when it feels too much.
+
+#### It's fine to say "I don't know"
+No one knows everything and we don't expect you to. It's fine to say you don't know something when asked. When you don't know something it's fine to ask questions and for help. You might not be alone in not understanding something so asking questions to the wider team in meetings or open channels can often help everyone.
+
+#### No blame
+We know mistakes and errors happen but instead of placing blame we encourage the reporting of them and take an active approach to learning from them when they happen.
+
+
+### Next steps
+
+Links to areas this is a pre-req for, other elements in a series, or other topics.
+
+
+
+Further resources
+----------------------------
+
+Links to other resources, and what they're useful for explaining.
```

Comment:
We should include links to articles/talks about culture and how we can all play our part in adhering to the principles listed above; I'll see if I can fish some out (other recommendations welcome)

Reply:
👍 agree - any links welcome